src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -151,6 +155,35 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
               <FormHelperText error>{errors.vendorId?.message}</FormHelperText>
             </FormControl>
           </Grid>
+          <Grid item container xs={6} spacing={2}>
+            <Grid item xs={12}>
+              <FormControl fullWidth>
+                <FormLabel>Expense Type</FormLabel>
+                <Controller
+                  name="expenseTypeId"
+                  control={control}
+                  render={({ field: { onChange, value } }) => (
+                    <Select
+                      onChange={(newValue) => onChange(newValue.target.value)}
+                      value={value}
+                      error={!!errors.expenseTypeId}
+                    >
+                      {allExpenseTypes.map((expenseType) => (
+                        <MenuItem key={expenseType.expenseTypeId} value={expenseType.expenseTypeId}>
+                          {expenseType.name}
+                        </MenuItem>
+                      ))}
+                    </Select>
+                  )}
+                />
+                <FormHelperText error>{errors.expenseTypeId?.message}</FormHelperText>
+              </FormControl>
+            </Grid>
+            <Grid item xs={12}>
+              <FormLabel>Total Cost</FormLabel>
+              <Typography variant="h6">${calculatedTotalCost}</Typography>
+            </Grid>
+          </Grid>
```

Comment:
for this part the only layout change we want is to swap refund source and expense type so that expense type is on top